core/domain/state_domain.py

```diff
@@ -1925,6 +1927,285 @@ def validate(self) -> None:
                 self.duration_secs)
 
 
+def validate_rte_tags(
```

Comment:
Should this go in html_validation_service considering this is not tied to State domain objects?

Reply:
Okay I will update that.

Reply:
Oh, I remember I will not be able to import these functions from the html_validation_service file because it will give me this listing error `Importing file named "*_service*" from module "core.domain*" in module "*core.domain.*_domain" is prohibited.`
Thanks

Reply:
Ah, I see. In that case, can we move it inside html_cleaner -- I see we have "get_rte_components" there which is related to RTE.

Reply:
Done